src/lib.rs

```diff
@@ -16,13 +16,14 @@ use nom::{
     branch::alt,
     bytes::complete::{tag, take_while, take_while1},
     character::complete::hex_digit1,
-    character::complete::{alpha1, alphanumeric1, char, digit1},
+    character::complete::{alpha1, alphanumeric1, char, digit1, satisfy},
```

Comment:
Can we add hex_digit1 here & get rid of the extra line?

Reply:
Done.

For the record, these import lines are automatically produced by the editor when I use symbols. If I notice I'll merge/cleanup/etc.